### DISCUSSION TITLE
Updated: Lab 1-html-css information

### DISCUSSION THREAD

**Discussion by Nortalle** [web-classroom/web-classroom.github.io#46] [Announcements]
Hi @web-classroom/web-vgx-students,

I updated the description for the "1-html-css" lab as follows:

> For the lab, do not forget to push your modifications to GitHub. The GitHub
> Actions will tell you if you passed all the tests.
> 
> You do not have to publish your website on GitHub Pages; it is not possible for
> this lab. Sorry for the confusion.
> 
> The grading will be done by the GitHub Actions; if you pass all the tests, you
> can be eligible for the full grade, but if we see some code that is too
> complicated, we can reduce the grade. You should not need more than one
> `position: absolute` and you do not need to use `position: relative` for this
> lab.
> 
> The lab is due on the 12st of March at 22:00.

You can see the change on the [`README.md`](https://github.com/web-classroom/heig-vd-web-course). And the commit is [here](https://github.com/web-classroom/heig-vd-web-course/commit/8b22535bbc572f0c692cf9c813cf231e76cdd65b)

**Comment by evarayHEIG** [comment]
Hi, thanks for the information! What about `position:fixed` ? Should we also avoid it?

**Reply by hadrylouis** [reply]
Hi Eva, 

Position fixed can be used (for the footer for example) although you can do without (with flexboxes only). 

Hope this helps

**Reply by evarayHEIG** [reply]
Ok, thank you for the answer.

**Comment by Yahia1018** [comment]
Hi,

Could it be possible for the next labos to have fixed directives from the beginning ?

Because it is a big waste of time to come back to the labo after days (I already have forgotten my method) to modify in consequence...

Thanks.

**Reply by Yahia1018** [reply]
Moreover, if no one had told me, I wouldn't have known, because we don't receive any notifications...

**Reply by hadrylouis** [reply]
Hello @Yahia1018,

You won't be penalized if you use absolute or relative positions in your code. @Nortalle and I simply noticed that many of you were using these directives and we wanted to let you know that you can do without. Using flexbox makes code simpler and cleaner, in our opinion.

However, if we find that you're abusing these directives (relative and absolute) and over-complicating your code, we may have to withdraw some points.

In addition, you should have received an email allowing you to join a Github team called web-vgx-students. This team allows you to receive notifications as soon as we mention @web-vgx-students in a discussion. On the other hand, you can use @web-vgx-teachers to have us notified. If you haven't received any notification, then either you haven't joined this team or you've disabled notifications from Github.

Hope this answers your questions.
